eng/create-announcement-draft.sh

```diff
@@ -9,20 +9,24 @@ print-help ()
   echo "--template            The base template to use for the announcement"
   echo "--channel             Release channel, e.g. '8.0'"
   echo "                      Note that when 6.0/7.0, dotnet/installer is targeted, otherwise dotnet/dotnet"
+  echo "--release             Name of the release, e.g. '8.0.0.preview-1'"
```

Comment:
```suggestion
  echo "--release             Programmatic name of the release, e.g. '8.0.0.preview-1'"
```

Reply:
@mthalman check this - I noticed the help is missing hints for values where they're needed and I asked Bing to add those for me:

```
print-help ()
{
  echo "Uses a template to draft a release announcement and outputs it into a file."
  echo
  echo "Options:"
  echo "--template=FILE       The base template to use for the announcement"
  echo "--channel=CHANNEL     Release channel, e.g. '8.0'"
  echo "                      Note that when 6.0/7.0, dotnet/installer is targeted, otherwise dotnet/dotnet"
  echo "--release=RELEASE     Name of the release, e.g. '8.0.0.preview-1'"
  echo "--release-name=NAME   Human readable name of the release, e.g. '.NET 8 Preview 1'"
  echo "--sdk-version=VERSION The .NET SDK version that is being released"
  echo "--runtime-version=VERSION The .NET runtime version that is being released"
  echo "--tag=TAG             The release tag, e.g. v8.0.0-preview.1"
  echo "--prerelease          (Optional) Whether this is a preview release"
  echo "--help, -h            (Optional) print this help message and exit"
  echo
}
```